File: performance-comparison/benchmarks/micro/clojure/float_arith.clj
(let [n (Long/parseLong (first *command-line-args*))
     result (loop [i 0 a 1.0 b 1.0]
               (if (>= i n) (+ a b)
                   (recur (inc i)
                          (+ (* a 1.0000001) (Math/sqrt b))
                          (+ (* b 0.9999999) (Math/sqrt a)))))]
  (printf "%.6f%n" result))

File: performance-comparison/benchmarks/micro/python/float_arith.py
import sys, math

n = int(sys.argv[1]) if len(sys.argv) > 1 else 1000000
a, b = 1.0, 1.0
for _ in range(n):
    a, b = a * 1.0000001 + math.sqrt(b), b * 0.9999999 + math.sqrt(a)
print(f"{a + b:.6f}")
